### ISSUE TITLE
Issue with "Test your skills: Functions": (short summary here please)

### ISSUE THREAD

**Issue by eakholod** [mdn/content#14794]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Test_your_skills:_Functions

#### What information was incorrect, unhelpful, or incomplete?

https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Functions
It provided a lot of information about functions. Unfortunately, it did not provide enough information for the test my skills.
#### Specific section or headline?
Test Your Skills Function 2 has a lot of requirements that we did not go over in the article based on functions.

#### What did you expect to see?
I was expecting to have questions and problems based on what I read and was doing in my active learning windows.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/javascript/building_blocks/test_your_skills_colon__functions`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Test_your_skills:_Functions
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/javascript/building_blocks/test_your_skills_colon__functions/index.md
* Last commit: https://github.com/mdn/content/commit/672bace59f60c4931e5cdde8eb32919c924bd191
* Document last modified: 2022-03-14T00:26:56.000Z

</details>

**Comment by wbamberg** [comment]
Oh, I saw the context for this at https://discourse.mozilla.org/t/test-your-skills-functions-learn-web-development-mdn/95602. I agree with you, this is not at all a fair test IMO.

@mikoMK , what do you think? Should we replace this with an exercise that is more self-contained?

To be honest I feel the same way about exercise 1, a bit. I'm not sure that "getting a random element from an array" is a thing we can expect learners to know without Googling it. Maybe we could provide that as part of the guidance for the exercise?

**Comment by mikoMK** [comment]
> Should we replace this with an exercise that is more self-contained?

@wbamberg Yes, I agree. That would probably make the exercise less frustrating. We have a short canvas example in the functions lesson before, but I think that's not enough to make it a fair exercise.

> To be honest I feel the same way about exercise 1, a bit.

I think exercise 1 is fine. In the lesson about functions we introduce the reader to `Math.random()` and we use and explain the following example (https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Functions#functions_versus_methods):
```
function random(number) {
  return Math.floor(Math.random()*number);
}
```
I think this one is close enough to exercise 2 to be enjoyable.

**Comment by eakholod** [comment]
@mikoMK Thank you for look my issue and being able to work it out. I am not familiar with the process but should I be waiting for it to be fixed or you will approach it differently?

**Comment by mikoMK** [comment]
Since we don't know how long it will take until there is a new exercise, I would recommend to either:
* Try solving task 2 with the additional hints I gave you in the forum
* Skip task 2 and continue with task 3

If you have more questions I'm happy to help you in the forum. This issue is fine as it is now. Nothing more to do for you here. 🙂

**Comment by mikoMK** [comment]
Yes, I meant exercise 1. Sorry for the confusion.

**Comment by Josh-Cena** [comment]
I believe this has been fixed by our recent test your skills restructure.
